kit/media/src/UIImage/hooks/useFlippedImageIfRtl.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { I18nManager, ImageStyle, StyleProp } from 'react-native';
+
+/**
+ * Flips images on the Y-axis in RTL mode.
+ * @param style
+ * @returns the same `style` with rotateY style in RTL mode.
+ */
+export function useFlippedImageIfRtl(
+    style: StyleProp<ImageStyle> | undefined,
+): StyleProp<ImageStyle> | undefined {
+    const isRTL = React.useMemo(() => I18nManager.getConstants().isRTL, []);
+
+    const flipImageIfRtlStyle: StyleProp<ImageStyle> = isRTL
+        ? {
+              transform: [
+                  {
+                      rotateY: '180deg',
```

Comment:
If the purpose is to flip the image horizontally than it should be:
```suggestion
                      scaleX: -1,
```